app/services/importer/scraper/workday_cookie_fetcher.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module Importer
+  module Scraper
+    class WorkdayCookieFetcher < ApplicationTask
+      include Capybara::DSL
+
+      def initialize(details)
+        @create_account_url = details[:create_account_url]
```

Comment:
Do we need so many variables in the initialise method?

Reply:
@create_account_url doesn't seem to be used (except for processable ... might be a good reason)

Reply:
Removed @create_account_url for now.